app/controllers/statistics_controller.rb

```diff
@@ -0,0 +1,17 @@
+class StatisticsController < ApplicationController
+  def count_metric
+    count_metric_snapshot = MetricSnapshot.where(code: params[:metric_code]).count
+  	total_configuration = MetricConfiguration.all.count
+
+    if(total_configuration == 0)
+      metric_percentage = 0
+    else
+      metric_percentage = (count_metric_snapshot.to_f / total_configuration.to_f) * 100
```

Comment:
We added the if/else statement because total_configuration can be zero, and that would lead to an mathematical error. But, if it is best, we can use a ternary operator in total_configuration.